modules/gapi/src/executor/gstreamingexecutor.hpp

```diff
@@ -195,6 +195,8 @@ class GStreamingExecutor final
 
     void wait_shutdown();
 
+    cv::GTypesInfo out_info;
```

Comment:
I believe it should start with `m_` like other fields but please fix that LATER.